.github/workflows/ci.yml

```diff
@@ -120,26 +56,26 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
       - name: Setup dart
-        uses: dart-lang/setup-dart@v1
+        uses: dart-lang/setup-dart@v1.4
       - name: Install dependencies
         run: dart pub get
       - name: Run build_runner
         run: dart run build_runner build --delete-conflicting-outputs
       - name: Run tests
-        run: dart test --coverage=coverage
+        run: dart test --coverage=./coverage --concurrency=4 --test-randomize-ordering-seed random
       - name: Convert code coverage
         # Needs to be adapted to collect the coverage at all platforms if platform specific code is added.
         if: ${{ always() && matrix.os == 'ubuntu-latest' }}
         working-directory: /
```

Comment:
`working-directory: /` 
remove this line not needed anymore

Reply:
It was done